pros/serial/devices/vex/v5_device.py

```diff
@@ -829,7 +830,9 @@ def get_device_status(self):
     def get_system_status(self) -> SystemStatus:
         from semantic_version import Version
         logger(__name__).debug('Sending ext 0x22 command')
-        if self.query_system_version().system_version < Version('1.0.13-0'):
+        sysVersion = self.query_system_version().system_version
+        if (self.SystemVersion.Product == self.SystemVersion.Product.BRAIN and sysVersion < Version('1.0.13-0')) or \
+         (self.SystemVersion.Product == self.SystemVersion.Product.CONTROLLER and sysVersion < Version('1.0.0-0.70')):
```

Comment:
```suggestion
        if (self.SystemVersion.Product == self.SystemVersion.Product.BRAIN and sysVersion in Spec('<1.0.13')) or \
         (self.SystemVersion.Product == self.SystemVersion.Product.CONTROLLER and sysVersion in Spec('<1.0.0-0.70')):
```

this is how we've been doing it elsewhere (see `V5Device.can_compress`)